content/Education and Kits/Science Journal App/Upload-the-Science-Journal-firmware.md

```diff
@@ -22,9 +23,13 @@ Boards supported by the Science Journal App require special firmware to connect.
 
 4. Click on **Examples** in the left-most sidebar to open the Examples tab.
 
-   - If you're using Nano 33 BLE Sense, search for "Nano33BLESenseFirmware" and open the  the result in the ARDUINO_SCIENCEJOURNAL library.
+   - If you're using the Nano RP2040 Connect, search for "Arduino_ScienceKitCarrier" and open the "ScienceJournal" example in the ARDUINO_SCIENCEKITCARRIER library.
```

Comment:
This is if you are using the RP2040 with the Science Kit R3 (Science Carrier R3). If you are using the board by itself, it's using the sketch from the Arduino_ScienceJournal library.